storage/emulator_test.sh

```diff
@@ -32,23 +32,40 @@ if [ "$minor_ver" -lt "$min_minor_ver" ]; then
 fi
 
 export STORAGE_EMULATOR_HOST="http://localhost:9000"
+export STORAGE_EMULATOR_HOST_GRPC="localhost:8888"
 
 DEFAULT_IMAGE_NAME='gcr.io/cloud-devrel-public-resources/storage-testbench'
 DEFAULT_IMAGE_TAG='latest'
 DOCKER_IMAGE=${DEFAULT_IMAGE_NAME}:${DEFAULT_IMAGE_TAG}
 CONTAINER_NAME=storage_testbench
 
-# Get the docker image for the testbench
-docker pull $DOCKER_IMAGE
-
-# Start the testbench
 # Note: --net=host makes the container bind directly to the Docker host’s network, 
 # with no network isolation. If we were to use port-mapping instead, reset connection errors 
 # would be captured differently and cause unexpected test behaviour.
 # The host networking driver works only on Linux hosts.
 # See more about using host networking: https://docs.docker.com/network/host/
-docker run --name $CONTAINER_NAME --rm --net=host $DOCKER_IMAGE &
+DOCKER_NETWORK="--net=host"
+if [ `go env GOOS` == 'darwin' ]; then
+    DOCKER_NETWORK="-p 9000:9000 -p 8888:8888"
```

Comment:
Do the retry conformance tests pass on mac with this?

Reply:
Not sure why, I think it's something to do with how errors differ when the networking is different - we see an EOF with the port forwarding; when it's on the same network, it's a connection error (which is what we are expecting). 

Either way, these tests weren't working using docker on mac before; imo it's fine as long as it works on linux. I would just add a comment mentioning that we expect the retry conformance tests to fail on darwin since we are specifically changing the network here.

Reply:
Ok, good idea. I've added such a comment, thanks for the review.